#! /bin/sh /usr/share/dpatch/dpatch-run
## 08_wireshark-desktop-menu.dpatch by  <fred@candle>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: change Icon and Categories for desktop file

@DPATCH@
diff -urNad trunk~/wireshark.desktop trunk/wireshark.desktop
--- trunk~/wireshark.desktop	2008-02-27 20:59:35.000000000 +0100
+++ trunk/wireshark.desktop	2008-03-12 21:21:00.000000000 +0100
@@ -68,10 +68,11 @@
 Comment[fi]=Verkkoliikenne analysaattori
 Comment[fr]=Analyseur de trafic réseau
 Comment[sv]=Nätverkstrafikanalysator
-Icon=wireshark.png
+Icon=hi48-app-wireshark.png
 TryExec=wireshark
 Exec=wireshark
 Terminal=false
 # Categorie entry according to version 1.0 of
 # http://www.freedesktop.org/wiki/Specifications/menu-spec
-Categories=System;Monitor;GTK;
+Categories=GNOME;Network;
+Encoding=UTF-8
